pkgs/tools/misc/dialog/default.nix

```diff
@@ -35,11 +40,11 @@ stdenv.mkDerivation (finalAttrs: {
     "install${lib.optionalString withLibrary "-full"}"
   ];
 
-  meta = with lib; {
+  meta = {
     homepage = "https://invisible-island.net/dialog/dialog.html";
     description = "Display dialog boxes from shell";
-    license = licenses.lgpl21Plus;
-    maintainers = with maintainers; [ AndersonTorres spacefrogg ];
+    license = lib.licenses.lgpl21Plus;
+    maintainers = with lib.maintainers; [ AndersonTorres spacefrogg ];
```

Comment:
You don't need to change meta around, it just increases the diff

Reply:
I  decline, because of the harms of nested `with`s.